Source/Core/arrayRemoveDuplicates.js

```diff
@@ -0,0 +1,89 @@
+/*global define*/
+define(['./defaultValue', './defined', './DeveloperError', './Math'], function(defaultValue, defined, DeveloperError, CesiumMath) {
```

Comment:
Can you run `npm run sortRequires` to fix the formatting for this.  It will also open a few other files for edit (which is OK, just submit them too).